SportsResults.frockett/SportsResults.frockett/SportsResultsService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Hosting;
+
+namespace SportsResults.frockett;
+
+internal class SportsResultsService : BackgroundService
+{
+    private readonly EmailService emailService;
+    private readonly ScraperService scraperService;
+
+    public SportsResultsService(EmailService emailService, ScraperService scraperService)
+    {
+        this.emailService = emailService;
+        this.scraperService = scraperService;
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+    {
+        DateTime startTime = DateTime.Now;
+        DateTime sendTime = new DateTime(startTime.Year, startTime.Month, startTime.Day, 16, 15, 0);
+        if (startTime > sendTime)
+        {
+            sendTime = sendTime.AddDays(1);
+        }
+
+        TimeSpan startupDelay = sendTime - startTime;
+        await Task.Delay(startupDelay, stoppingToken);
+
+        try
+        {
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                var results = await scraperService.GetScoresAsync();
+                var table = EmailComposer.BuildHtmlTable(results);
+                EmailService.SendEmail(table);
+
+                sendTime = sendTime.AddDays(1);
```

Comment:
What if the user wants to change the time at which the email will be sent? :smiley:

Reply:
Sorry, I don't understand what you're getting at here. Weren't we meant to make something akin to a service that would run constantly in the background on a server, not something like an app for many users? In that case, it could be changed in the code. I don't see the other projects in the repo allowing the user to adjust the time in another way.

Reply:
To answer specifically, it would be changed up here, where sendTime is initialized 

![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.SportsResults/assets/119839633/e23bd3d8-3680-4c9a-b4db-912bc46eb5e5)

Reply:
If I'm being dense and not understanding what you mean, then let me know 😅

Reply:
I asked this question specifically because when I ran the project for the first time without really going through the source code I was waiting for the email to be delivered and it never got delivered so I looked through the code to only to find out that the delivery time was set in the past. 

Time I ran the service: 23:05
sendTime: 16:27 ??

As you can see this is in the past.

Reply:
Ah I get what you mean now, yeah I set the time based on when I was working on it